graphql/resolvers.go

```diff
@@ -444,6 +444,7 @@ func (r *mutationResolver) SpawnHost(ctx context.Context, spawnHostInput *SpawnH
 }
 
 func (r *mutationResolver) EditSpawnHost(ctx context.Context, editSpawnHostInput *EditSpawnHostInput) (*restModel.APIHost, error) {
+	var err error
```

Comment:
To fix lint, you'll want to remove this line and instead declare `var v *Volume` before line 499, and use `v, err =` instead of := (to avoid redeclaring err in the if block)

Reply:
Thanks!